docs/maintenance.md

```diff
@@ -0,0 +1,20 @@
+# Maintenance procedure
+
+This page describes how to update tenet regularly.
+
+## Regular Update
+
+1. Update `go.mod`.
+2. Update `k8s-version` used in GitHub Actions. They are kind node tags.
+3. Update `KUBERNETES_VERSION` in `Makefile.versions`.
+4. Update Go & Ubuntu versions if needed.
+5. Check for new software versions using `make version`. You may be prompted to login to github.com.
```

Comment:
Since running `make version` without the required binaries will cause it to fail, I felt it would be a better to describe the setup procedure.